Fixup for 595419.

Document blkiomon option -d (--dump-lldd).
Add drv_data mask description to blktrace man page.

Signed-off-by: Edward Shishkin <edward@redhat.com>
---

--- blktrace-1.0.1.orig/blkiomon.c
+++ blktrace-1.0.1/blkiomon.c
@@ -599,6 +599,7 @@
 	"-I <interval>       | --interval=<interval>\n" \
 	"[ -h <file>         | --human-readable=<file> ]\n" \
 	"[ -b <file>         | --binary=<file> ]\n" \
+	"[ -d <file>         | --dump-lldd=<file> ]\n" \
 	"[ -D <file>         | --debug=<file> ]\n" \
 	"[ -Q <path name>    | --msg-queue=<path name>]\n" \
 	"[ -q <msg queue id> | --msg-queue-id=<msg queue id>]\n" \
--- blktrace-1.0.1.orig/doc/blkiomon.8
+++ blktrace-1.0.1/doc/blkiomon.8
@@ -7,7 +7,7 @@
 
 .SH SYNOPSIS
 .B blkiomon \-I \fIinterval\fR [ \-h \fIfile\fR ] [ \-b \fIfile\fR ]
-[ \-D \fIfile\fR ] [ \-Q \fIpath_name\fR
+[ \-d \fIfile\fR ] [ \-D \fIfile\fR ] [ \-Q \fIpath_name\fR
 \-q \fImsg_queue_id\fR \-m \fImsg_id\fR ] [ \-V ]
 .br
 
@@ -50,6 +50,13 @@
 Binary output file. Use '\-' for stdout.
 .RE
 
+\-d \fIfile\fR
+.br
+\-\-dump-lldd=\fIfile\fR
+.RS
+Output file for data emitted by low level device driver.
+.RE
+
 \-D \fIfile\fR
 .br
 \-\-debug=\fIfile\fR
--- blktrace-1.0.1.orig/doc/blktrace.8
+++ blktrace-1.0.1/doc/blktrace.8
@@ -172,6 +172,8 @@
 \fIwrite\fR: write traces
 .br
 \fInotify\fR: trace messages
+.br
+\fIdrv_data\fR: additional driver specific trace
 .RE
 
 
